tests/Keywords/CucumberKeywordsTest.php

```diff
@@ -11,23 +11,27 @@
 namespace Tests\Behat\Gherkin\Keywords;
 
 use Behat\Gherkin\Keywords\CucumberKeywords;
+use Behat\Gherkin\Keywords\KeywordsInterface;
 use Behat\Gherkin\Node\StepNode;
 use Symfony\Component\Yaml\Exception\ParseException;
 use Symfony\Component\Yaml\Yaml;
+use Tests\Behat\Gherkin\Filesystem;
 
 class CucumberKeywordsTest extends KeywordsTestCase
 {
-    protected function getKeywords()
+    protected static function getKeywords(): KeywordsInterface
     {
         return new CucumberKeywords(__DIR__ . '/../Fixtures/i18n.yml');
     }
 
-    protected function getKeywordsArray()
+    protected static function getKeywordsArray(): array
     {
-        return Yaml::parse(file_get_contents(__DIR__ . '/../Fixtures/i18n.yml'));
+        $data = Filesystem::readFile(__DIR__ . '/../Fixtures/i18n.yml');
+
+        return Yaml::parse($data);
```

Comment:
you could use `Yaml::parseFile()` instead (which also mentions the file name in the error message, as it knows it)